### PR TITLE
Add --list-validator-indices flag

### LINKED ISSUE #8404: List all validator id's
# 🚀 Feature Request

### Description

I would like a straight forward command to output the ?index id's? of your validator(s). Not sure if _index id_ is the correct word for it but basically the validator id number that is shown in the second column here: https://mainnet.beaconcha.in/validators/leaderboard

### Describe the solution you'd like

`--list-validator-id   --all`
Requests a dump of available id's, while --all selects all. Specifiying --pubkey 0x1  0x2  0x3  0x4 can be useful too.

### Describe alternatives you've considered

Beaconcha.in works, if you don't have a lot of validators. However, I don't think we should force users to a website when our own data surely got to be available on our own staking machines.

### PR DESCRIPTION
**Pull request by eklm**
**What type of PR is this?**

Feature

**What does this PR do? Why is it needed?**
Adds an option to list validator ids: 
`
./validator accounts list --list-validator-indices
`
**Which issues(s) does this PR fix?**

Fixes #8404

**Other notes for review**

### CODE REVIEW

File: validator/accounts/accounts_list.go [hunk 1]
```diff
@@ -221,3 +233,26 @@ func listRemoteKeymanagerAccounts(
...
+	}
+	req := &ethpb.MultipleValidatorStatusRequest{PublicKeys: pks}
+	resp, err := client.MultipleValidatorStatus(ctx, req)
+	if err != nil {
+		return errors.Wrap(err, "could not request validator indices")
+	}
+	fmt.Println(au.BrightGreen("Validator indices:").Bold())
+	for _, idx := range resp.Indices {
+		if idx != math.MaxUint64 {
+			fmt.Println(idx)
```
Comment (rauljordan) [inline validator/accounts/accounts_list.go:254]: Can we print them alongside the first 8 characters of each public key for easier identification?

**Comment by rauljordan** [comment]
Really nice addition and simple PR! Just added a small recommendation on the printed output

File: validator/accounts/accounts_list.go (hunk 1 above)
Comment (eklm) [reply validator/accounts/accounts_list.go]: Sure, added public keys to the command output.

**Review by rauljordan — APPROVED** [review]

**Comment by terencechain** [comment]
Sorry, I have a nitpick. How about adding a column as separator?

Currently:
```
Validator indices:
0x8fc602ac 100021
0xaec0e753 100023
0xb01010a5 100025
0xa4882167 100026
0xb64322a4 100027
0xa1879b9e 100028
0xaceafd8e 100020
0x9026fa37 100022
0xaeb2c6eb 100024
```

Will look nicer:
```
Validator indices:
0x8fc602ac: 100021
0xaec0e753: 100023
0xb01010a5: 100025
0xa4882167: 100026
0xb64322a4: 100027
0xa1879b9e: 100028
0xaceafd8e: 100020
0x9026fa37: 100022
0xaeb2c6eb: 100024
```

**Review by terencechain — APPROVED** [review]
Thanks!

**Comment by makultar** [comment]
I cant stop being impressed by all coders who work with this. Is there anything you CANT DO?????????

**Comment by makultar** [comment]
Your brain just see solutions to every problem. Im amazed.

**Pull request merged**